pkg/database/migrations.go

```diff
@@ -2087,8 +2087,7 @@ func (db *Database) Migrations(ctx context.Context) []*gormigrate.Migration {
 			ID: "00093-AddClaimMeanAgeToRealmStats",
 			Migrate: func(tx *gorm.DB) error {
 				return multiExec(tx,
-					`ALTER TABLE realm_stats ADD COLUMN IF NOT EXISTS code_claim_mean_age BIGINT NOT NULL DEFAULT 0`,
-					`ALTER TABLE realm_stats DROP COLUMN IF EXISTS codes_claimed_age_distribution`) // result of previous mistake
+					`ALTER TABLE realm_stats ADD COLUMN IF NOT EXISTS code_claim_mean_age BIGINT NOT NULL DEFAULT 0`)
```

Comment:
Is this intended?

Reply:
Yea I think it's confusing to leave in. It only got deployed to dev